cpp/src/cider/exec/nextgen/operators/AggregationNode.cpp

```diff
@@ -82,7 +83,43 @@ std::vector<int8_t> initOriginValue(context::AggExprsInfoVector& exprs_info) {
   int8_t* raw_memory = origin_vector.data();
   for (const auto& info : exprs_info) {
     switch (info.agg_type_) {
-      case SQLAgg::kSUM:
+      case SQLAgg::kSUM: {
```

Comment:
this function is too long, you can split it into multi small functions according to agg_type